local/build.go

```diff
@@ -89,7 +102,7 @@ func (s *local) build(ctx context.Context, project *types.Project, opts map[stri
 	return err
 }
 
-func (s *local) buildImage(ctx context.Context, service types.ServiceConfig, contextPath string) build.Options {
+func (s *composeService) buildImage(service types.ServiceConfig, contextPath string) build.Options {
```

Comment:
nit: maybe rename this to something like `getBuildOptions`, I found `buildImage` misleading